tests/fixtures/locales-with-missing-keys/shitcoiner.json

```diff
@@ -0,0 +1,22 @@
+{
+  "shitcoin": {
+    "description": "An offensive term for other cryptocurrencies different from bitcoin",
+    "glosary": {
+      "pumpAndDump": {
+        "description": "A clever marketing strategy to make a cryptocurrency look like a good investment."
+      },
+      "exitScam": {
+        "description": "Something that won't happen to my project, I swear."
+      }
+    }
+  },
```

Comment:
Maybe use a less controversial topic or wording for these fixtures? 😅 If this gem is ever used by outside devs that support some other crypto, or even new devs here, I wouldn't want anyone to feel pushed away or something

Reply:
haha ok, I'll change that 😅